src/esp/bindings_js/modules/simenv_embind.js

```diff
@@ -192,6 +267,15 @@ class SimEnv {
     }
     return converted;
   }
+
+  convertVec3fToVector3(position) {
```

Comment:
Why doesn't the ES6 JS ellipsis syntax take care of this construction? Does it really need to be its own function? Alternatively, can't we embind the construct so it just takes in a JS  list as well?